Objects/genobject.c

```diff
@@ -269,13 +269,29 @@ gen_send_ex2(PyGenObject *gen, PyObject *arg, PyObject **presult,
 }
 
 PySendResult
-PyGen_Send(PyGenObject *gen, PyObject *arg, PyObject **result)
+PyIter_Send(PyObject *iter, PyObject *arg, PyObject **result)
```

Comment:
Hm, what function does the ceval look call now when in tracing?

Reply:
It calls `_PyObject_CallMethodIdOneArg(receiver, &PyId_send, v)`.

Reply:
Makes sense.